### PR TITLE
Created simple pre-loader

### PR DESCRIPTION
**Pull request by Re42**
fixes issue #32 by adding a preloader to the site:
live gh-pages for review:
https://re42.github.io/OpenCode-Collaborative/

...finally!

### CODE REVIEW

**Comment by aswinzz** [comment]
@Re42  can you try some other design?rest all is good!

**Comment by Anupam-dagar** [comment]
@Re42 since you are using a gif as loader, i would suggest finding good ones, there are a lot of choices in gif's for loader.

**Comment by Re42** [comment]
running
How about this?

**Comment by aswinzz** [comment]
nop

**Comment by Re42** [comment]
The issue says simple..

**Comment by pradeepgangwar** [comment]
@aswinzz ^^

**Comment by Re42** [comment]
running2

I credited, I guess that means its okay to use this?
it's from this repo: https://github.com/yangshun/awesome-spinners and hence under creative commons license.

**Comment by pradeepgangwar** [comment]
@aswinzz looks ok to me. ur views?

**Comment by aswinzz** [comment]
cool!

**Comment by aswanthkoleri** [comment]
@Re42 Resolve the conflicts ASAP.

**Comment by gauthamzz** [comment]
A rotating circle is enough. If you wanna do fancy, you should do something using the logo of opencode.

**Comment by Re42** [comment]
@aswinzz uploaded new gif w/o text, @pradeepgangwar resolved conflicts... everything okay for merge?

File: index.html [hunk 1]
```diff
@@ -492,18 +501,6 @@ <h4 class="contact_head">Mubaris NK</h4>
...
-
-
-
-
-
-  <script src="https://unpkg.com/vue"></script>
-  <script src="js/mentor.js"></script>
-  <script src="js/contestant.js"></script>
-  <script src="js/projects.js"></script>
-  <script src="js/issue-loader.js"></script>
```
Comment (pradeepgangwar) [inline index.html]: You deleted this script (probably by mistake) and issue loader does not work in your fork. Please check this that you don't delete thing not related to PR. :)

File: index.html [hunk 2]
```diff
@@ -534,16 +531,12 @@ <h4 class="contact_head">Mubaris NK</h4>
 
 
 
-
-
 </body>
 <script src="https://unpkg.com/vue"></script>
 <script src="js/mentor.js"></script>
 <script src="js/contestant.js"></script>
 <script src="js/projects.js"></script>
```
Comment (pradeepgangwar) [inline index.html]: Add scripts before closing body tag, not after. :)

**Comment by pradeepgangwar** [comment]
Reast looks good

File: index.html (hunk 1 above)
Comment (Re42) [reply index.html]: made both requested changes, sorry for the trouble, merge?

File: index.html [hunk 3]
```diff
@@ -491,18 +500,11 @@ <h4 class="contact_head">Mubaris NK</h4>
...
-  <script src="https://unpkg.com/vue"></script>
-  <script src="js/mentor.js"></script>
-  <script src="js/contestant.js"></script>
-  <script src="js/projects.js"></script>
-  <script src="js/issue-loader.js"></script>
-
+   <script src="https://unpkg.com/vue"></script>
+   <script src="js/mentor.js"></script>
+   <script src="js/contestant.js"></script>
+   <script src="js/projects.js"></script>
```
Comment (pradeepgangwar) [inline index.html]: these scripts reference has been repeated ... look below they are already there.

Comment (Re42) [reply index.html]: That was why I deleted them in the first place, but I get it, your change was just to ass the issue script and not all the ones I deleted.

**Comment by Re42** [comment]
made changes

**Comment by aswinzz** [comment]
Make it center aligned. Rest all LGTM.

**Comment by gauthamzz** [comment]
Its running for a very long time. This is bad. make issues to improve speed of website.

**Comment by gauthamzz** [comment]
this commit when merging make sure its squashed 🔸

**Comment by Anupam-dagar** [comment]
LGTM :+1:

**Comment by Re42** [comment]
Centered it...

**Comment by gauthamzz** [comment]
Its taking too long, make it like. If it is taking more than 2 seconds. Get out of the loader

**Comment by Re42** [comment]
I made the change for it to exit in 2 seconds, but I think it broke something else?
I don't see the gif anymore...

**Comment by Re42** [comment]
Done, thanks!

**Pull request merged**